### DISCUSSION TITLE
Does this sort the items into the layout you set up?

### DISCUSSION THREAD

**Discussion by Massivefro** [LopyMine/SpecificSlots#5] [General]
Had a question about the layout. FIrst I can't get to get anything to 'load' after a save it. But once I do does this automatically sort everything into those specific slots? Or is this just aesthetic?

**Comment by LopyMine** [comment]
This mod is more visually tuned, but it also has some sorting features, for example, if you click right mouse button with shift to the item, your item will (possibly) end up in the config slot

**Comment by Massivefro** [comment]
Ah okay. I was looking for a mod to replace the old ‘Inventory Tweaks’ mod that allowed me to sort items into specific slots. Being able to die or rearrange then immediately snap everything back into their correct slots at the push of a button was superb. But no one has recreated anything similar & then I saw your mod. I didn’t understand the point of just having a visual aid so I was a bit confused. You have any idea how difficult it is to implement a feature like that? Allowing a ‘sort’ option but have ‘specific slots’ designated to a specific item or ‘type’ of item?S. WilliamsonOn Jan 13, 2024, at 3:20 PM, LopyMine ***@***.***> wrote:﻿
This mod is more visually tuned, but it also has some sorting features, for example, if you click right mouse button with shift to the item, your item will (possibly) end up in the config slot
